docs/e2e.md

```diff
@@ -51,7 +54,14 @@ Tests are defined in [/testing/e2e/suites/](../testing/e2e/suites/). Add any new
 
 ## GitHub Runner
 
-todo: will fill in details when I add GitHub workflows to run new e2e
+We use GitHub workflows to run and require passing E2E tests on every PR. [.github/workflows/ok-to-test.yaml](../.github/workflows/ok-to-test.yaml), [.github/workflows/pr-validate.yaml](../.github/workflows/pr-validate.yaml), [.github/workflows/pr-validate-fork.yaml](../.github/workflows/pr-validate-fork.yaml), [.github/workflows/e2ev2.yaml](../.github/workflows/e2ev2.yaml), and [.github/workflows/e2ev2-provision-test.yaml](../.github/workflows/e2ev2-provision-test.yaml) chain together to provide this.
```

Comment:
beyond "chain together" can you add some description about what each one does?

Reply:
done